ci/test.lua

```diff
@@ -141,6 +141,49 @@ local function load_test_config(config_file)
     return config
 end
 
+local function run_expect_func(func, ...)
+    local args = {...}
+    local saved_printerr = dfhack.printerr
+    local printerr_called = false
+    dfhack.printerr = function(msg) printerr_called = true end
+    return dfhack.with_finalize(
+        function() dfhack.printerr = saved_printerr end,
+        function()
+            local ret = {func(table.unpack(args))}
+            if printerr_called then
+                return {false,
```

Comment:
Is this supposed to be a table? Most `expect` functions return multiple values instead of a table.

Reply:
it's a table so that we capture the variable number of comment values. It's used in `wrap_expect` below. [1] is true/false, [2] and on are comments.